### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hrittikhere**
@mikenikles can you please take a look into this? I have updated Azure docs with changes

### CODE REVIEW

**Comment by akosyakov** [comment]
@jeanp413 we changed to 3000?

**Comment by hrittikhere** [comment]
image
I remember it to be ` 3000 ` when I wrote the documentation for the first time. But it shows ` 8000 ` now

**Comment by akosyakov** [comment]
I'm not sure it maybe fixed already in insiders to be 3000 again.

**Comment by hrittikhere** [comment]
Okay @akosyakov. Please let me know then I will revert 8000 and just keep the token part.

**Comment by jeanp413** [comment]
@akosyakov not yet, I'll change it today.
@hrittikhere I'll change the default port to 3000 so please revert that change

**Comment by hrittikhere** [comment]
@jeanp413 @akosyakov updated the ports.

File: guides/azure-vm/README.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ cd openvscode-server-v$SERVER_VERSION-linux-x64
 
 1. Navigate to the Overview pane for the VM
 1. Copy the "Public IP address"
-1. Paste the IP address in a new browser tab and add `:3000`, i.e. `http://52.251.44.195:3000`
+1. Visit the URL printed in your terminal. Just going to port 3000 won't work, because VS Code requires that you provide a uniquely generated security token to prevent unauthorized access.
+1. Replace 'localhost' from the IP address on the terminal to your "Public IP address" in a new browser tab , i.e. ` http://localhost:3000/?tkn=f83f27f2-3e75-47a8-be84-6a8396d1491b` would become `http://52.251.44.195:3000/?tkn=f83f27f2-3e75-47a8-be84-6a8396d1491b`
```
Comment (bigint) [inline guides/azure-vm/README.md:59]:
```suggestion
1. Replace 'localhost' from the IP address on the terminal to your "Public IP address" in a new browser tab, i.e. ` http://localhost:3000/?tkn=f83f27f2-3e75-47a8-be84-6a8396d1491b` would become `http://52.251.44.195:3000/?tkn=f83f27f2-3e75-47a8-be84-6a8396d1491b`
```

**Pull request merged**